data/frame_type.go

```diff
@@ -156,3 +162,41 @@ const KindTimeSeries FrameTypeKind = "timeseries"
 //
 // [Data Plane Numeric Kind]: https://github.com/grafana/grafana-plugin-sdk-go/blob/main/data/contract_docs/numeric.md
 const KindNumeric FrameTypeKind = "numeric"
+
+// FrameType is a 2 number version (Major / Minor).
+type FrameTypeVersion [2]uint
```

Comment:
Alternatively we could use a standard semver string.  I prefer that slightly since it is well known and has a clear way to indicate "alpha/beta/dev" api status.  

The `[2]uint` approach would match thema, but unless it is using the same constraints under the hood, that may just cause confusion.

Parsing the numbers from semver is not ideal, however the existence of well known libraries should make that better.